ChangeLog.md

```diff
@@ -22,6 +22,9 @@
 
 - Defines an `evalFresh` handler for `Control.Carrier.Strict.FreshC`, taking the initial value. ([#267](https://github.com/fused-effects/fused-effects/pull/267))
 
+- Adds a `Control.Effect.Exception` module, lifting & re-exporting operations & types from `Control.Exception` using the new definition of `Lift`. ([#306](https://github.com/fused-effects/fused-effects/pull/306))
```

Comment:
As mentioned in the other comment (which isn't as of this writing visible in this diff), I think we should move this to `fused-effects-exceptions`, and make it the canonical interface to `Control.Exception`'s functionality.

Reply:
Whoops, forgot to remove this note! Good catch.